examples/with-redux-observable/README.md

```diff
@@ -29,24 +29,56 @@ npm run dev
 
 
 ### The idea behind the example
-Example is a page that renders information about Star-Wars characters. It fetches new character  
-every 3 seconds having the initial character fetched on a server.
+
+This example is a page that renders information about Star-Wars characters. It
+fetches new character every 3 seconds having the initial character fetched on
+a server.
 
 Example also uses `redux-logger` to log every action.
 
 ![demo page](demo.png)
 
-The main problem with integrating Redux, Redux-Observable and Next.js is probably making initial requests
-on a server. That's because it's not possible to wait until epics are resolved in `getInitialProps` hook. 
+The main problem with integrating Redux, Redux-Observable and Next.js is
+probably making initial requests on a server. That's because, the
+`getInitialProps` hook runs on the server-side before epics have been made available by just dispatching actions.
+
+However, we can access and execute epics directly. In order to do so, we need to
+pass them an Observable of an action and they will return an Observable:
+
+```js
+static async getInitialProps({ store, isServer }) {
+  const resultAction = await rootEpic(
+    of(actions.fetchCharacter(isServer)),
+    store
+  ).toPromise(); // we need to convert Observable to Promise
+  store.dispatch(resultAction)};
+```
+
+Note: we are not using `AjaxObservable` from the `rxjs` library; as of rxjs
+v5.5.6, it will not work on both the server- and client-side. Instead we call
+the default export from
+[universal-rx-request](https://www.npmjs.com/package/universal-rx-request) (as
+`ajax`).
```

Comment:
Someday in the future I'd like to figure out how to do this without `universal-rx-request`, overriding the `XMLHTTPRequest` used by `ajax`. I think it's possible with Rx, I haven't tried recently though.